OpenEdX/RouteController.swift

```diff
@@ -36,15 +36,14 @@ class RouteController: UIViewController {
             }
         } else {
             DispatchQueue.main.async {
-                self.showAuthorization()
+                self.showStartupScreen()
             }
         }
     }
     
-    private func showAuthorization() {
+    private func showStartupScreen() {
         let controller = UIHostingController(
-            rootView: SignInView(viewModel: diContainer.resolve(SignInViewModel.self)!)
-        )
+            rootView: StartupView(viewModel: diContainer.resolve(StartupViewModel.self)!))
         navigation.viewControllers = [controller]
         present(navigation, animated: false)
     }
```

Comment:
Providing a feature flag for this feature would be good, as many users don't need this functionality by default.

Reply:
@volodymyr-chekyrta  If we want to move this behind a feature flag, then we have to maintain many things, like the conditional hide/show of the navbar on the sign-in and register screens, and also the hide/show of the register button on the sign-in screen.

I guess we can have the startup screen by default because it will be an additional screen with more information about the product.

If you still want me to move this behind a feature flag, please let me know. I'll make all the relevant changes.

Reply:
@volodymyr-chekyrta Have you thought about this how are are going with it?

Reply:
In the Android PR we have this feature flag.
https://github.com/openedx/openedx-app-android/pull/86
So, it would be great if we could follow the same strategy and feature flags for both platforms.

Reply:
Done